docs/5.api/rpc/maintenance-windows.md

```diff
@@ -0,0 +1,126 @@
+---
+id: maintenance-windows
+title: Maintenance Windows
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The RPC API enables you to query future maintenance windows for a specific validator in current epoch
+
+---
+
+## Maintenance windows {#maintenance-windows}
+
+> The maintenance windows for a specific validator are future block height ranges in current epoch, in which the validator do not need produce block or chunk
```

Comment:
I'd also mention that if the provided account is NOT a validator - then it will return the range from 'now' to the end of the epoch.